### PR TITLE
2.8.3 Release Notes

### PR DESCRIPTION
**Pull request by michaeljmarshall**
### Motivation

Here are the release notes for 2.8.3. This is a work in progress. Please let me know if you have any recommendations.

### Modifications

* Add notes to the `website`

### CODE REVIEW

**Comment by Anonymitaet** [comment]
Hi @urfreespace we do not need to add [release-notes.md](https://github.com/apache/pulsar/pull/14632/files#diff-0d837aba6b2ed3e09c3696ef3e68ebbae94f73a10332435fc4ed98173457433a) to `site2/website-next`, you'll sync changes from `site2/website` to site2/website-next` regularly, correct？

**Comment by michaeljmarshall** [comment]
@Anonymitaet - do we have a documented process for where to contribution documentation right now? Thanks!

**Comment by Anonymitaet** [comment]
@michaeljmarshall this is a "temporary" status. I'll document the workflow once it gets finalized. All website content has been synced to https://github.com/apache/pulsar-site/tree/main/site2
@urfreespace is deleting `pulsar/site2/website-next` to avoid confusion https://github.com/apache/pulsar/pull/14640

**Comment by michaeljmarshall** [comment]
@Anonymitaet - thank you for letting me know. I'll rebase this PR.

File: site2/website/release-notes.md [hunk 1]
```diff
@@ -1,6 +1,207 @@
 
 ## Apache Pulsar Release Notes
 
+### 2.8.2
+
+#### 2022-03-10
+
+### Important Notices
+
+- Fixed detecting number of NICs in EC2 [#14252](https://github.com/apache/pulsar/pull/14252). In the event that Pulsar cannot determine the NIC speed from the host, please set `loadBalancerOverrideBrokerNicSpeedGbps`.
```
Comment (Anonymitaet) [inline site2/website/release-notes.md:10]: suggest using [present tense imperative](https://web.mit.edu/course/21/21.guide/m-impera.htm#:~:text=The%20imperative%20mood%20is%20used,base%20form%20of%20the%20verb.)

File: site2/website/release-notes.md [hunk 2]
```diff
@@ -1,6 +1,207 @@
...
+
+### Test
+- Improved 9 flaky tests.
+
+### Tools
+- Fix NPE in cmdTopics [13450](https://github.com/apache/pulsar/pull/13450)
+- [pulsar-perf] Support listenerThreads configuration. [12892](https://github.com/apache/pulsar/pull/12892)
+- [docs] Fix doc for pulsar-admin bookies cmd [12542](https://github.com/apache/pulsar/pull/12542)
+
+### Topic-Policy
```
Comment (Anonymitaet) [inline site2/website/release-notes.md:188]:
```suggestion
### Topic policy
```

File: site2/website/release-notes.md [hunk 3]
```diff
@@ -1,6 +1,207 @@
...
+
+### Transaction
+- Fix topicTransactionBuffer handle null snapshot (#12758) [14510](https://github.com/apache/pulsar/pull/14510)
+- Fix transaction system topic create in loop [12889](https://github.com/apache/pulsar/pull/12889)
+- Fix transaction system topic create in loop. [12749](https://github.com/apache/pulsar/pull/12749)
+
+### Websocket
+- Fix ``ClassCastException`` when user create ``MultiTopicReader``. [14316](https://github.com/apache/pulsar/pull/14316)
+
+### Miscellaneous
```
Comment (Anonymitaet) [inline site2/website/release-notes.md:200]:
```suggestion
### Dependency
```

File: site2/website/release-notes.md [hunk 4]
```diff
@@ -1,6 +1,207 @@
...
+- Remove unnecessary Pulsar Client usage from Pulsar Proxy [13836](https://github.com/apache/pulsar/pull/13836)
+
+### Python Client
+- Apply clang-format check for python wrapper [13418](https://github.com/apache/pulsar/pull/13418)
+
+### Schema
+- Fix parse BigDecimal [14019](https://github.com/apache/pulsar/pull/14019)
+- Fix pulsar use json or avro primitive schema. [12886](https://github.com/apache/pulsar/pull/12886)
+
+### Sql
```
Comment (Anonymitaet) [inline site2/website/release-notes.md:176]:
```suggestion
### SQL
```

**Review by Anonymitaet — COMMENTED** [review]
@michaeljmarshall do not forget to write 2.8.3 tech blog https://pulsar.apache.org/blog/

**Review by lhotari — APPROVED** [review]
LGTM

**Comment by michaeljmarshall** [comment]
> @michaeljmarshall do not forget to write 2.8.3 tech blog https://pulsar.apache.org/blog/

@Anonymitaet - I won't be able to complete a blog for the release. Our wiki currently lists this as an optional part of the release process: https://github.com/apache/pulsar/wiki/Release-process#18-write-a-blog-post-for-the-release-optional.

**Pull request merged**